index.html

```diff
@@ -1,75 +1,102 @@
-        </section>
-
-        <section class="bottom-content">
-
-            <div class="text-container">
-                <h2>That's</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>No</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-            <div class="text-container">
-                <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
-            </div>
-
-        </section>
-        
-        <footer>
-            <nav>
-                <a href="#">Home</a>
-                <a href="#">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
-        </footer>
-    </div><!-- container -->        
+	<header>
+		<nav>
+			<div class="logo"><img alt="lambda logo" src="img/lambda-black.png"></div>
+			<ul>
+				<li>
+					<a href="#">Home</a>
+				</li>
+				<li>
+					<a href="about.html">About</a>
+				</li>
+				<li>
+					<a href="#">Products</a>
+				</li>
+				<li>
+					<a href="#">Blog</a>
+				</li>
+				<li>
+					<a href="#">Contact</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<div class="jumboparent">
+		<div class="jumbochild"><img alt="jumbo" src="img/jumbo.jpg"></div>
+	</div>
+	<div class="container">
+		<section class="top-content">
+			<div class="text-container">
+				<h2>The Future</h2>
+				<p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+			</div>
+			<div class="text-container">
+				<h2>The Past</h2>
+				<p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+			</div>
+		</section>
+		<section class="middle-content">
+			<h2>Why Did It Have To Be Boxes...</h2>
+			<div class="boxes">
+				<div class="box" id="box1">
+					Box 1
+				</div>
+				<div class="box" id="box2">
+					Box 2
+				</div>
+				<div class="box" id="box3">
+					Box 3
+				</div>
+				<div class="box" id="box4">
+					Box 4
+				</div>
+				<div class="box" id="box5">
+					Box 5
+				</div>
+				<div class="box" id="box6">
+					Box 6
+				</div>
+				<div class="box" id="box7">
+					Box 7
+				</div>
+				<div class="box" id="box8">
+					Box 8
+				</div>
+				<div class="box" id="box9">
+					Box 9
+				</div>
+				<div class="box" id="box10">
+					Box 10
+				</div>
```

Comment:
So this works, but I would use the id's sparingly. The was to do this in convention is to add another class. The reason being is because all of the boxes look like the class `box` and then you are only changing the color of the box. So the best way to do that is to add another class with a new color.

For Example

```<div class="box" id="box1">Box 1</div>```
Should be
```<div class="box box1">Box 1</div>```
And 
```<div class="box" id="box2">Box 2</div>```
Should be 
```<div class="box box2">Box 2</div>```

Notice how I just made another class instead of an id. Now in your css you can just style it like normal. 

Your way absolutely work! It's just as best practices, use another class to override styles on one class. You'll see why once we get to JavaScript 😉